envs/environment.docs.yaml

```diff
@@ -9,7 +9,7 @@ dependencies:
   - python<=3.7
   - pip
   - pypsa>=0.17.1
-  - atlite=0.0.3
+  - atlite=0.2.1
```

Comment:
Intentionally fixed or should this also be `atlite>=0.2.1` ?

Are we missing `mamba` in the doc env?

Reply:
Correct, atlite>=0.2.1 is better.
Regarding mamba, we install it beforehand (at least in the CI) and install the env via mamba.

Reply:
Ok.